docs/user-guide/cli-using-initializing-team-configuration.md

```diff
@@ -1,56 +1,97 @@
 
-    **Example:** List all data sets under your user ID:
     ```
-    zowe zos-files list data-set "MY.DATASET.*"
+    zowe plugins install @zowe/cics-for-zowe-cli
     ```
 
-    A list of data sets is returned. You successfully configured Zowe CLI to access a z/OSMF instance.
+    **Note:** If the `zowe.config.json` file has not yet been created in the `.zowe` directory, see [Creating team profile configuration files](#creating-team-profile-configuration-files).
+
+2. Run the `zowe config init --global-config` or `zowe config auto-init --global-config` command.
+
+    This adds a plug-in profile to the configuration file in the `.zowe` root directory.
+
+3. Open the `zowe.config.json` file and confirm the plug-in profile is included.
     
-    If the CLI returns an error message, verify that you have access to the target system. Examine your configuration files in a text editor to verify that the information you entered is correct.
+    In the example from Step 1, the profile information displays similarly to the example below:
 
+    ```
+    "cics": {
+            "type": "cics",
+            "properties": {
+                "port": 123
+            },
+            "secure": []
+        }
+    ```
+
+    The plug-in profile has been successfully added to the `zowe.config.json` file in the `.zowe` root directory.
 
-**Important:** After the configuration files are in place (either by using the `zowe config init` command or by manually creating the files), the now-deprecated zowe profiles commands no longer function. Zowe CLI returns errors when you attempt to use deprecated profile commands.
+    **Note:** To add plug-in profiles to a configuration file in the current working directory, enter the base command without the option: `zowe config init`.
 
 ## Connecting profiles to API Mediation Layer
 
-If you are using the API Mediation Layer, you can set up your `zowe.config.json` file to automatically access the services that are registered to the API ML by issuing the following command:
+If you are using the API Mediation Layer, set up the `zowe.config.json` file to automatically access the services that are registered to the API ML and support Single Sign-On.
 
-```
-zowe config auto-init
-```
+1. Run the following command:
+
+    ```
+    zowe config auto-init --global-config
+    ```
+
+2. Respond to subsequent CLI prompts with the following information:
+
+    - The host name and port to your API ML instance.
+    
+    - Your username and password.
 
-After you issue the command, Zowe CLI prompts you to specify the following information:
+    A default profile is added to the configuration file in the `.zowe` root directory.
 
-- The host name and port to your API ML instance
-- Your username and password
+**Note:** To add a profile to a configuration file in the current working directory, enter the base command without the option: `zowe config init`.
```

Comment:
same as teh previous comment about the option name

Reply:
At first I thought adding the option code would make things hard to read, but I've now changed my mind. Updated to add the option code.